lib/jekyll/url.rb

```diff
@@ -107,15 +108,17 @@ def generate_url_from_drop(template)
         value = "" if value.nil?
         replacement = self.class.escape_path(value)
 
-        match.sub(":#{winner}", replacement)
-      end.squeeze("/")
+        match.sub!(":#{winner}", replacement)
+      end
```

Comment:
Notes:
- Calling `String#squeeze` here is redundant *with respect to the bigger picture (`Jekyll::URL.new.to_s`)* because the result from this method is passed onto `Jekyll::URL#sanitize_url` that removes extra `"/"` from input strings.
- The local `match` variable value is not further required. So it can be mutated.